__tests__/app.test.js

```diff
@@ -81,5 +81,27 @@ describe('/api/articles/:article_id', () => {
           });
       });  
 });
+describe('/api/articles', () => {
+    test('GET - status: 200 - get all the articles but body', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((res) => {
+            res.body.articles.forEach((article) => {
```

Comment:
Can add an expect to test that you get all of the articles back - i.e. do an expect(res.body.articles.length).toBe(however many articles there are).